modules/images/webp-uploads/readme.txt

```diff
@@ -0,0 +1,59 @@
+=== WebP Uploads ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.1
+Tested up to:      6.1
+Requires PHP:      5.6
+Stable tag:        1.0.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images, webp
+
+Creates WebP versions for new JPEG image uploads if supported by the server.
+
+== Description ==
+
+This plugin adds WebP support for media uploads within the WordPress application.
```

Comment:
Technically not a blocker, but since this is pretty much the only module / plugin that has a UI to it, we still may want to provide at least a bit more explanation in 2-3 short paragraphs. Things to potentially highlight:
* WebP images will be generated only for new uploads
* By default, only WebP images will be generated for JPEG uploads, only the original uploaded file will still exist as JPEG
* There is a checkbox in `Settings > Media` that can change the behavior though to generate _both_ JPEG and WebP for every sub-size (noting again that this will only affect newly uploaded images, i.e. after making that change)

Reply:
@felixarntz (cc @mukeshpanchal27 ) this is ready for re-review following changes pushed in recent commit(s).